src/adapter/src/coord/ddl.rs

```diff
@@ -1067,7 +1067,11 @@ impl Coordinator {
                                 Connection::AwsPrivatelink(_) => {
                                     new_aws_privatelink_connections += 1
                                 }
-                                Connection::Csr(_) | Connection::Ssh(_) | Connection::Aws(_) => {}
+                                // TODO(roshan): Implement limits for MySQL
```

Comment:
I'm a bit nervous that we'll forget something if it only exists in TODO comments. I suggest we both leave a TODO in the code as you have here and also adding whatever it is to be done in the work item list in this issue https://github.com/MaterializeInc/materialize/issues/15901

I added these ones already